exporters/trace/gcp_exporter/internal/gcp_exporter_test.cc

```diff
@@ -0,0 +1,107 @@
+#include "gtest/gtest.h"
+#include <stdlib.h>
+#include "../gcp_exporter.h"
+#include "opentelemetry/sdk/trace/simple_processor.h"
+#include "opentelemetry/sdk/trace/tracer_provider.h"
+#include "opentelemetry/trace/provider.h"
+#include "opentelemetry/core/timestamp.h"
+#include "google/devtools/cloudtrace/v2/tracing_mock.grpc.pb.h"
+
+using namespace testing;
```

Comment:
using declaration for a whole namespace is discouraged as it can lead to namespace collision, it would be better to have using declarations for individual objects.